src/server/api.ts

```diff
@@ -249,6 +250,22 @@ function setupRoutes() {
         }
     );
 
+    // New API access to endpoint, use for log data insertion
+    app.post(
+        '/api/v1/logUpload',
+        ensureLoggedIn({ throw: true }),
```

Comment:
For testing purposes, we can comment this line & remove the dataUpload.ts part, since it's not useful until the client-side file upload is done.

For example, do the following locally for testing:
```curl -d @someFile.json http://localhost:3001/api/v1/logUpload```